rest/polygon.go

```diff
@@ -13,6 +13,7 @@ type Client struct {
 	ReferenceClient
 	TradesClient
 	SnapshotClient
+	VXClient
```

Comment:
this needs to be `VX VXClient`. what you're doing here is go's weird form of inheritance which i don't want in this case